tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -0,0 +1,453 @@
+                            </devCustomizationsCustomControls:DevDriveCardBody>
+                        </Grid>
+                    </Grid>
+                </ListViewItem>
+            </DataTemplate>
+
+
+            <!--- Template for optimization suggestion cards on the dev insights page. -->
+            <DataTemplate x:Key="HorizontalCardForDevDriveOptimizerInsightsPage" x:DataType="devViewModels:DevDriveOptimizerCardViewModel">
+                <ListViewItem
+                    DataContext="{Binding}"
+                    IsSelected="{Binding IsSelected, Mode=OneWay}"
+                    HorizontalAlignment="Stretch"
+                    AutomationProperties.ItemStatus="Loaded">
+                    <Grid
+                        Style="{StaticResource HorizontalCardRootForSetupTargetFlow}">
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <devCustomizationsCustomControls:DevDriveOptimizerCardBody
+                                Grid.Row="1"
+                                CacheToBeMoved="{Binding CacheToBeMoved, Mode=OneWay}"
+                                ExistingCacheLocation="{Binding ExistingCacheLocation, Mode=OneWay}"
+                                ExampleLocationOnDevDrive="{Binding ExampleLocationOnDevDrive, Mode=OneWay}"
+                                EnvironmentVariableToBeSet="{Binding EnvironmentVariableToBeSet, Mode=OneWay}">
+                            </devCustomizationsCustomControls:DevDriveOptimizerCardBody>
+                        </Grid>
+                    </Grid>
+                </ListViewItem>
+            </DataTemplate>
+
+            <!--- Template for optimized cards on the dev insights page. -->
+            <DataTemplate x:Key="HorizontalCardForDevDriveOptimizedInsightsPage" x:DataType="devViewModels:DevDriveOptimizedCardViewModel">
+                <ListViewItem
+                    DataContext="{Binding}"
+                    IsSelected="{Binding IsSelected, Mode=OneWay}"
+                    HorizontalAlignment="Stretch"
+                    AutomationProperties.ItemStatus="Loaded">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="*" />
+                            <RowDefinition Height="*" />
+                        </Grid.RowDefinitions>
+                        <controls:SettingsCard Grid.Row="1" Header="{x:Bind CacheMoved, Mode=OneWay}">
+                            <controls:SettingsCard.Description>
+                                <TextBlock TextWrapping="Wrap" Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}">                            
+                                        <Run>The environment variable </Run>

```

Comment:
Did your content person sign off on these strings?

Reply:
Yes. The figmas were created in coordination with the design team.